### DISCUSSION TITLE
How to get current `basePath` using app directory?

### DISCUSSION THREAD

**Discussion by remorses** [vercel/next.js#49312] [App Router]
Previously you could do
```ts
import { router } from 'next/router'

router.basePath
```

How can you do the same with `next/navigation`?

**Comment by Codeshark-NET** [comment]
In my case, I found that simply removing the `router.basePath` everywhere seemed to resolve properly while setting the `basePath` accordingly in the `next.config.js`, see here: https://nextjs.org/docs/app/api-reference/next-config-js/basePath 🙌
